src/push-notifications.js

```diff
@@ -59,6 +59,13 @@ export async function init(config) {
     deviceStateStore,
     endpointOverride,
   });
+
+  const deviceExists = deviceId !== null;
+  if (deviceExists) {
+    await instance._updateDeviceMetadata();
```

Comment:
What happens to this await if the promise from `instance._updateDeviceMetadata` gets rejected?

Will it still return the `instance`?